Form/Extension/HelpMessageTypeExtension.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Linio\DynamicFormBundle\Form\Extension;
+
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+
+class HelpMessageTypeExtension extends AbstractTypeExtension
+{
+    /**
+     * @return string
+     */
+    public function getExtendedType()
+    {
+        return FormType::class;
+    }
+
+    /**
+     * @param OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefined(array('help'));
+    }
+
+    /**
+     * @param FormView $view
+     * @param FormInterface $form
+     * @param array $options
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        $view->vars['help'] =  isset($options['help']) ? $options['help'] : "";
```

Comment:
Remove the extra space in the assignment, and also use simple quotes in the empty string.